backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
Drawing list is catching the chat signal for new revisions on existing drawings ok, but new drawings are ignored
![24-09-10 05-35-35](https://github.com/user-attachments/assets/18aef75c-6a4a-4ef3-95d4-1d56503e965b)